serving/src/main/resources/templates/single_featureset_pit_join.sql

```diff
@@ -1,7 +1,7 @@
 WITH union_features AS (SELECT
   uuid,
   event_timestamp,
-  NULL as {{ featureSet.name }}_v{{ featureSet.version }}_feature_timestamp,
+  NULL as {{ featureSet.project }}_{{ featureSet.name }}_v{{ featureSet.version }}_feature_timestamp,
```

Comment:
I know it's easier to construct these queries using the fully qualified names, but we should ideally have a way to just return the feature names to users, no project, feature set, or version. Perhaps out of scope for this PR.

Reply:
That's easily doable if we commit to never querying for features of the same name

Reply:
I'm okay with that if you are

Reply:
I am super ok with that. We can always add flags later that allow them to prefix results with `project` or `feature set`

Reply:
done